src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -121,18 +121,23 @@ public void removeSession(Name name, Index sessionIndex) {
 
     /**
      * Returns student with {@code name}.
-     * @param name must be unique and exact.
-     * @return Student
+     * The student with the exact name must already exist in the address book.
      */
     public Student getStudentWithName(Name name) {
         requireNonNull(name);
         return students.getStudentWithName(name);
     }
 
+    /**
+     * Returns true if a student with this name exists in the unique student list
```

Comment:
I think this might violate the 'explain WHAT and WHY, not HOW' @ [textbook](https://nus-cs2103-ay2021s2.github.io/website/se-book-adapted/chapters/codeQuality.html#guideline-comment-minimally-but-sufficiently).

Reply:
Good point! Can you give an example of how you would phrase it? I'm not sure how best to change it.

Reply:
Honestly, i don't have a clear idea of how to phrase as well 🤣 
One way is to remove comments, since it's pretty self-explanatory (as per textbook), maybe someone else has a better suggestion? @AY2021S2-CS2103T-T11-1/developers